acceptance/indy-cli-batches/AS-03-01-trustee.batch

```diff
@@ -9,8 +9,8 @@ did new seed=TestTrustee200000000000000000000 metadata="test trustee 2"
 did new seed=TestTrustee300000000000000000000 metadata="test trustee 3"
 did new seed=TestSteward100000000000000000000 metadata="test steward 1"
 did new seed=TestSteward200000000000000000000 metadata="test steward 2"
-did new seed=TestTrustAnchor00000000000000000 metadata="test trust anchor 1"
-did new seed=TestTrustAnchor10000000000000000 metadata="test trust anchor 2"
+did new seed=TestEndorser00000000000000000 metadata="test endorser 1"
```

Comment:
The seed's length needs to be 32. Please add missing 0s.

Reply:
fixed